src/cli/cli.go

```diff
@@ -94,6 +117,8 @@ func Handle() {
 	loglevel, logfile := logger.PreloadLoggingFlags()
 	ctx, log := logger.Seed(ctx, loglevel, logfile)
 
+	ctx = clues.AddAll(ctx, "corso_version", version.CurrentVersion())
```

Comment:
Does this get logged in every log/error message?

Reply:
Yes

Reply:
Yeah - I think it'll be overkill. Having it in the log file should be good enough (maybe we put in CLI output as well as log file?) 

We should also print on the command line the log file full-path we are using (this will help the user identify which log file to look at)